8.0/mysql-variables.txt

```diff
@@ -4,7 +4,7 @@ character_set_database	utf8mb4
 character_set_filesystem	binary
 character_set_results	utf8mb4
 character_set_server	utf8mb4
-character_set_system	utf8
+character_set_system	utf8mb3
```

Comment:
Per 
- https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_character_set_system
- https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_character_set_system 

`character_set_system` is always `utf8` (default). As such, we should just drop it altogether.